### ISSUE TITLE
[Catalog] Unchecking a card doesn't work in TalkBack mode

### ISSUE THREAD

**Issue by pubiqq** [material-components/material-components-android#3176]
https://user-images.githubusercontent.com/82187521/213261681-1d2e2a61-93a2-499d-b526-566d3e173e36.mp4

**Material Library version:** 1.9.0-alpha01
**Device:** Pixel 4a 5G (Android 13)

**Comment by paulfthomas** [comment]
Hi @pubiqq, 2 accessibility actions have been added in order to select/deselect. See https://github.com/material-components/material-components-android/commit/4bd5e619fbe742c6606e28c8c54b3da2cd3816fc.

**Comment by pubiqq** [comment]
It seems the issue description wasn't specific enough, so I've clarified it.

**Comment by pubiqq** [comment]
@paulfthomas, it seems the issue description wasn't specific enough, so I've clarified it.

**Comment by pubiqq** [comment]
@paulfthomas, the description was misunderstood, so I've clarified it. Could you take another look at the issue?

**Comment by paulfthomas** [comment]
@pubiqq I think there is an issue with SelectionTracker and TalkBack, the deselection seems to work though with a "short" long press. The accessibility actions were added to make it more usable in the current state. The SelectionTracker issue is being investigated separately.
